auth-api/src/auth_api/services/org.py

```diff
@@ -800,6 +800,11 @@ def search_orgs(search: OrgSearch, environment):  # pylint: disable=too-many-loc
                         if include_invitations and org.invitations
                         else []
                     ),
+                    "members": (
+                        MembershipSchema(exclude=("org", "user.contacts")).dump(org.members, many=True)
```

Comment:
org already has a relationship in the model for members, we can just pull the data and use the schema conversion